### DISCUSSION TITLE
Add ENABLE_AUTOPAUSE=TRUE to existing Docker.

### DISCUSSION THREAD

**Discussion by Yamanipanuchi** [itzg/docker-minecraft-server#957] [Q&A]
Running an Unraid server. I want to add the AutoPause feature to an already running instance. I cant seem to discern how you do this?

I am going to guess it is most likely so simple I am just over looking it.

**Comment by Yamanipanuchi** [comment]
Seems as you ask the question the solution presents itself. Looks as if in Unraid you add them and Variables and now knowing that, It makes sense how the docker command works at the command line.

Let me know if I have this wrong?

**Reply by itzg** [reply]
I'm not familiar with the Unraid aspect, but yes you're right that it is mostly about adding the environment variables to the container declaration.
